drp/api/posts.py

```diff
@@ -173,18 +173,16 @@ def post(self):
         files = request.files.getlist('files')
         names = request.form.getlist('names')
 
-        if title is None or summary is None or content is None:
```

Comment:
I would perhaps suggest recovering these checks before merging. They are probably not vital, but I still think they are useful as a sanity check. As the title, summary and content fields are always visible on the frontend, it wouldn't probably make much sense to send a request without one of them.

Reply:
Yeah I think we want to make it more restrictive, not less. Ideally we should check that they are all not `None` and also not empty.

Reply:
Alright, I will add them back in